tests/app/helpers.py

```diff
@@ -7,11 +7,10 @@
 
 from app import create_app
 from app import login_manager
-from unittest import TestCase
 
 
-class BaseApplicationTest(TestCase):
-    def setUp(self):
+class BaseApplicationTest(object):
+    def setup_method(self, method):
```

Comment:
(it also strikes me that a lot of the complex pytest fixture behaviours could be achieved through multiple inheritance but whatever)